src/static/js/AttributeManager.js

```diff
@@ -406,7 +406,7 @@ AttributeManager.prototype = _(AttributeManager.prototype).extend({
       hasAttrib = this.getAttributeOnSelection(attributeName);
     } else {
       const attributesOnCaretPosition = this.getAttributesOnCaret();
-      hasAttrib = _.contains(_.flatten(attributesOnCaretPosition), attributeName);
+      hasAttrib = [].concat(...attributesOnCaretPosition).includes(attributeName);
```

Comment:
does includes find within multidimensional array tho?  Afaik that's why I did ``concat(...`` is to flatten it down?

Reply:
Oh, I missed the flatten. It might improve readability to either add a comment or split it into two lines of code:

```suggestion
      const allAttribs = [].concat(...attributesOnCaretPosition);  // flatten
      hasAttrib = allAttribs.includes(attributeName);
```